test/functional/editor_controller_test.rb

```diff
@@ -11,9 +11,9 @@ def setup
     assert_redirected_to '/login'
   end
 
-  test "should get post form" do
+  test "should get legacy form" do
     UserSession.create(rusers(:bob))
-    get :post,
+    get :legacy,
```

Comment:
I see what you're doing here, and it makes sense. But I think we need to change the tests, or add new tests, so that the new `post` action is as well tested as the previous one was. Does that make sense? So, for each one you're converting to a legacy test, we should add an equivalent test for the new `post` action. 

Is that something you think you could try? Not every bit of HTML will be the same, but we can look at how the new `/post` page renders, and adjust the assertions to match the new one. I'm happy to offer input/review as you go -- you can start with just one test, not the whole set -- thank you!